src/soc_ifc/rtl/soc_ifc_subsystem_reg.rdl

```diff
@@ -216,3 +216,20 @@ reg {
             [br]TAP Access [in debug/manuf mode]: RO";
     field {sw=rw; swwel=soc_req; hw=r; } go[32] = 32'h0;
 } SS_GENERIC_FW_EXEC_CTRL[4];
+reg {
+    name = "External Staging Area Registers";
+     desc = "External staging area address for Caliptra Core. 64-bit address split into 2x32bit registers with [0] being the lower address and [1] being the upper address. Shall be in Caliptra crypto boundary. Lockable by EXTERNAL_STAGING_AREA_ADDRESS_LOCK
+             [br]Caliptra Access: RW
+             [br]SOC Access:      RW
+             Read-Only once locked by EXTERNAL_STAGING_AREA_ADDRESS_LOCK.";
+     field {sw=rw; hw=r; swwel;resetsignal = cptra_rst_b;} addr[32]=0x0;
+ } EXTERNAL_STAGING_AREA_ADDRESS[2];
```

Comment:
nit: the other addresses like this use a `_l` and `_h` address, rather than an array of size 2

Reply:
Fixed